src/search/components/query-editor.controller.js

```diff
@@ -168,8 +160,8 @@ function QueryEditorController(
         subNode.fieldType = fieldMapping[subNode.field].type || 'unknown';
 
         if (subNode.fieldType === 'date-range') {
-          subNode.lowerBound = subNode.lowerBound === '*' ?  '*' :  new Date(subNode.lowerBound);
-          subNode.upperBound = subNode.upperBound === '*' ? '*' :  new Date(subNode.upperBound);
+          subNode.lowerBound = subNode.lowerBound === '*' ?  '*' :  new Date(subNode.lowerBound.replace(/\\/g, ''));
+          subNode.upperBound = subNode.upperBound === '*' ? '*' :  new Date(subNode.upperBound.replace(/\\/g, ''));
```

Comment:
So you're stripping `\` from date string values here?

Reply:
Yes correct